RatScanner/ViewModel/SettingsVM.cs

```diff
@@ -148,6 +190,11 @@ public async Task SaveSettings() {
 		RatEye.Config.LogDebug = RatConfig.LogDebug;
 		RatScannerMain.Instance.HotkeyManager.RegisterHotkeys();
 
+		if (IsFloatingTooltipEnabled)
+		{
+			FloatingTooltip.Instance.Hide();

```

Comment:
To instantiate the FloatingTooltip Instance when needed and to skip it if it is disabled.